packages/graphback-cli/src/templates/resources/models/Tasks.graphql

```diff
@@ -0,0 +1,31 @@
+type Task {
+  id: ID!
+  version: Int
+  title: String!
+  description: String!
+  status: String!
+  creationMetadata: CreationMetadata
+  assignedTo: User
+}
+
+type CreationMetadata {
+   createdDate: String
+   createdBy: User
+ }
+
+type User {
+  id: ID!
+  firstName: String!
+  lastName: String!
+  title: String!
+  email: String!
+}
+
+## Custom queries and mutations
+type Query {
+  allTasks(first: Int, after: String): [Task],
```

Comment:
`allTasks` in pretty similar to the `findAll` method that's gonna be generated. Maybe switch to `allCompletedTasks`?